packages/suite/src/components/suite/modals/ReviewTransaction/components/OutputElement.tsx

```diff
@@ -123,63 +131,105 @@ const TruncateWrapper = ({
     children?: React.ReactNode;
 }) => (condition ? <Truncate>{children}</Truncate> : <>{children}</>);
 
-const OutputLine = ({
+const getFingerprint = (
+    tokens: Account['tokens'],
+    symbol: string | undefined,
+): string | undefined => {
+    if (!tokens) {
+        return undefined;
+    }
+
+    const token = tokens.find(token => token.symbol?.toLowerCase() === symbol?.toLowerCase());
+
+    return token?.name;
```

Comment:
Maybe if we could have some comment here or even better a link to documentation? On the first sight it looks hard to understand why method name is "getFingerprint" but it returns token name. Does token name equal fingerprint?

Reply:
✓